 <launch>
	<node pkg="raposang_odometry" name="raposang_odometry" type="raposang_odometry_node" output="screen">
	 
		<remap from="input_imu"          			to="/imu/data"/>
		<remap from="input_odo"     					to="/raposang/odometry"/>
		<remap from="output_pose"     				to="/raposang/odometry_pose"/>
		
	</node>		
	
<!--	<node pkg="tf" type="static_transform_publisher" name="tfodo" args="0 0 0 0 0 0 world raposang_odometry 100" /> 
	
 	<node pkg="rviz" type="rviz" name="rviz" /> -->
	
 </launch>
